src/lib_c/x86_64-windows-msvc/c/ntsysteminfo.cr

```diff
@@ -0,0 +1,54 @@
+require "c/winnt"
+require "c/win_def"
+require "c/int_safe"
+
+lib LibC
+  fun NTGetSystemInfo = GetSystemInfo(system_info : SystemInfo*)
+
+  PROCESSOR_ARCHITECTURE_AMD64   =      9
+  PROCESSOR_ARCHITECTURE_ARM     =      5
+  PROCESSOR_ARCHITECTURE_ARM64   =     12
+  PROCESSOR_ARCHITECTURE_IA64    =      6
+  PROCESSOR_ARCHITECTURE_INTEL   =      0
+  PROCESSOR_ARCHITECTURE_UNKNOWN = 0xffff
+
+  struct ProcessorInfo
+    wProcessorArchitecture : WORD
+    wReserved : WORD
+  end
+
+  union OEMProcessorInfo
+    dwOemId : DWORD
+    processorInfo : ProcessorInfo
+  end
+
+  struct SystemInfo
+    oemProcessorInfo : OEMProcessorInfo
+    dwPageSize : DWORD
+    lpMinimumApplicationAddress : LPVOID
+    lpMaximumApplicationAddress : LPVOID
+    dwActiveProcessorMask : DWORD_PTR
+    dwNumberOfProcessors : DWORD
+    dwProcessorType : DWORD
+    dwAllocationGranularity : DWORD
+    wProcessorLevel : WORD
+    wProcessorRevision : WORD
+  end
+
+  fun NTGetComputerNameExA = GetComputerNameExA(computer_name_format : ComputerNameFormat,
```

Comment:
Ditto, no `NT`.

Reply:
And use `GetComputerNameExW` for unicode compat.

Reply:
Fixed [here](https://github.com/crystal-lang/crystal/blob/b9b590442aa9d0afcce2cb16af0e65e8a846ae76/src/lib_c/x86_64-windows-msvc/c/sysinfoapi.cr#L31).